.travis.yml

```diff
@@ -19,11 +19,6 @@ stages:
 
 jobs:
   include:
-    - name: "Build npm cache"
```

Comment:
Found that the caching functionality was broken, so removed it.  In the next PR that I issue for this ticket, we're going to be using `npm ci` and lockfiles.  `npm ci` deletes the `node_modules` directory, which will defeat the purpose of caching anyway.

Reply:
Why are we splitting this up into separate PRs?

Also, the motivation to use `npm ci` is that it will always install only from the lockfile and thus be completely deterministic?

Reply:
I broke it into separate PRs because tackling both at once (npm _and_ ci + lockfiles) made it really hard to isolate what was causing a given failure.  Moving to "npm only" seemed like a reasonable way to bite off a smaller piece.

As for `npm ci` and lockfiles, yeah, the goal is to make sure that we're always using the same set of package versions.